src/AppLayout.tsx

```diff
@@ -10,26 +11,32 @@ export enum Widths {
 }
 
 function AppLayout() {
-    const [navigationOpen, setNavigationOpen] = useState(false);
-    const [navWidth, setNavigationWidth] = useState<Widths>(Widths.RAIL);
+    const [navigationConfig, setNavigationConfig] = useLocalStorage(
+        LocalStorageKeys.NAVIGATION_SETTINGS,
+        { open: true, width: Widths.FULL }
```

Comment:
We shouldn't be storing the actual width in the local storage. This prevents us from changing that without forcing some kind of update or asking folks to clear cache